### PR TITLE
[init] Refine long-summary for `az init`

### PR DESCRIPTION
**Pull request by zhoxing-ms**
---

Issue: https://github.com/Azure/azure-cli/issues/23283

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repository and upgrade the version in the pull request but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
refine az init

**Review by yonzhan — APPROVED** [review]

File: src/init/azext_init/_help.py [hunk 1]
```diff
@@ -10,4 +10,11 @@
 helps['init'] = """
     type: command
     short-summary: It's an effortless setting up tool for configs.
+    long-summary: |
+        This tool can help you quickly know and set up global configurations that are suitable for your current usage environment:
```
Comment (chasewilson) [inline src/init/azext_init/_help.py:14]:
```suggestion
        az init can help you quickly understand and set up global configurations that are suitable for your current usage environment:
```

File: src/init/azext_init/_help.py [hunk 2]
```diff
@@ -10,4 +10,11 @@
...
+        It supports setting bundles for interactive mode and automatic mode to help you quickly set the recommended configurations.
```
Comment (chasewilson) [inline src/init/azext_init/_help.py:15]:
```suggestion
        It supports setting bundles for interactive and automation scenarios to help you quickly set the recommended configurations.
```

File: src/init/azext_init/_help.py [hunk 3]
```diff
@@ -10,4 +10,11 @@
...
+        It also supports walk through style to help you get familiar with the common configurations and customize them one by one.
```
Comment (chasewilson) [inline src/init/azext_init/_help.py:16]:
```suggestion
        It also supports a walk-through experience to help you get familiar with the common configurations and customize them one by one.
```

File: src/init/azext_init/_help.py [hunk 4]
```diff
@@ -10,4 +10,11 @@
...
+
+        In addition, it's purpose is not to replace the "az config" command, but to reduce the common repeated configuration behaviors by supporting setting bindles.
```
Comment (chasewilson) [inline src/init/azext_init/_help.py:18]:
```suggestion
        The purpose of "az init" is not to replace the "az config" command, but to simplify common configurations by supporting setting bundles.
```

File: src/init/azext_init/_help.py [hunk 5]
```diff
@@ -10,4 +10,11 @@
...
+        The configuration set by "az init" can also be modified by "az config", because they modify the same configuration file.
```
Comment (chasewilson) [inline src/init/azext_init/_help.py:19]:
```suggestion
       While "az init" and "az config" modify the same configuration file, "az init" is aimed at an interactive, standardized approach to managing your configuration and "az config" is better at more granular control and setting configurations in automation scenarios.
```

**Review by chasewilson — CHANGES_REQUESTED** [review]
Each of these are just suggestions. Feel free to adjust as necessary :)

File: src/init/azext_init/_help.py [hunk 6]
```diff
@@ -10,4 +10,11 @@
 helps['init'] = """
     type: command
     short-summary: It's an effortless setting up tool for configs.
+    long-summary: |
+        az init can help you quickly understand and set up global configurations that are suitable for your current usage environment:
+        It supports setting bundles for interactive and automation scenarios to help you quickly set the recommended configurations.
+        It also supports a walk-through experience to help you get familiar with the common configurations and customize them one by one.
+
+        The purpose of "az init" is not to replace the "az config" command, but to simplify common configurations by supporting setting bundles.
+       While "az init" and "az config" modify the same configuration file, "az init" is aimed at an interactive, standardized approach to managing your configuration and "az config" is better at more granular control and setting configurations in automation scenarios.
```
Comment (zhoxing-ms) [inline src/init/azext_init/_help.py:19]:
```suggestion
        While "az init" and "az config" modify the same configuration file, "az init" is aimed at an interactive, standardized approach to managing your configuration and "az config" is better at more granular control and setting configurations in automation scenarios.
```

**Comment by zhoxing-ms** [comment]
@chasewilson Thank you for your contribution. They look clearer and much better 😊

**Review by chasewilson — APPROVED** [review]

**Comment by chasewilson** [comment]
@dbradish-microsoft, can you review this before merge? :)

File: src/init/azext_init/_help.py [hunk 7]
```diff
@@ -10,4 +10,11 @@
 helps['init'] = """
     type: command
     short-summary: It's an effortless setting up tool for configs.
```
Comment (dbradish-microsoft) [inline src/init/azext_init/_help.py:12]: short-summary: Set Azure CLI user configurations by responding to questions with multiple choice answers

long-summary: |
The Azure CLI offers `az init` to help you set global configurations through an interactive question-answer format.  Global settings are grouped into bundles that help you optimize your settings for scenarios such as "interaction" or "automation".  To familiarize yourself with common configurations and select available options, answer the step-by-step questions, and `az init` will do the rest!

The purpose of `az init` is not to replace the `az config` command, but to simplify the setting of configurations by offering an interactive question-answer experience.  While both `az inti` and `az config` modify the same configuration file, `az init` is designed to only set the most common configurations.  For more granular control of all available configurations, use `az config`.  For more information on configuring your Azure CLI environment, see https://docs.microsoft.com/en-us/cli/azure/azure-cli-configuration.

**Review by dbradish-microsoft — CHANGES_REQUESTED** [review]
This command is awesome!  We need to advertise it!!  

**Format assumption:**  We can use the backtick in the long-summary.

**Feedback on the interactive instructional sentences:**
Can we please consider using active voice (i.e. an understood "you") by rewriting these statements as follows?

"This option is used to enable/disable the progress bar for long running commands" becomes "Enable/disable the progress bar for long-running commands."

"This options to enable/disable colored syntax highlighting" becomes "Enable/disable colored syntax highlighting".  If you don't update this sentence, please correct the typo by changing it to read "This option is used too ...".  This same grammar correction is needed for `message recommendations`.

**Note!**  I really like that you took the time to articulate why our customers might want to select one option over another.

File: src/init/azext_init/_help.py (hunk 7 above)
Comment (dbradish-microsoft) [inline src/init/azext_init/_help.py:12]:
```suggestion
    short-summary: Set Azure CLI user configurations by responding to questions with multiple choice answers.
```

File: src/init/azext_init/_help.py [hunk 8]
```diff
@@ -10,4 +10,11 @@
...
+    long-summary: |
```
Comment (dbradish-microsoft) [inline src/init/azext_init/_help.py:13]: long-summary: |
The Azure CLI offers `az init` to help you set global configurations through an interactive question-answer format.  Global settings are grouped into bundles that help you optimize your settings for scenarios such as "interaction" or "automation".  To familiarize yourself with common configurations and select available options, answer the step-by-step questions, and `az init` will do the rest!

The purpose of `az init` is not to replace the `az config` command, but to simplify the setting of configurations by offering an interactive question-answer experience.  While both `az inti` and `az config` modify the same configuration file, `az init` is designed to only set the most common configurations.  For more granular control of all available configurations, use `az config`.  For more information on configuring your Azure CLI environment, see https://docs.microsoft.com/en-us/cli/azure/azure-cli-configuration.

Comment (dbradish-microsoft) [inline src/init/azext_init/_help.py:13]:
```suggestion
long-summary: |
The Azure CLI offers `az init` to help you set global configurations through an interactive question-answer format.  Global settings are grouped into bundles that help you optimize your settings for scenarios such as "interaction" or "automation".  To familiarize yourself with common configurations and select available options, answer the step-by-step questions, and `az init` will do the rest!

The purpose of `az init` is not to replace the `az config` command, but to simplify the setting of configurations by offering an interactive question-answer experience.  While both `az inti` and `az config` modify the same configuration file, `az init` is designed to only set the most common configurations.  For more granular control of all available configurations, use `az config`.  For more information on configuring your Azure CLI environment, see https://docs.microsoft.com/en-us/cli/azure/azure-cli-configuration.
```

**Comment by zhoxing-ms** [comment]
@dbradish-microsoft Thank you for putting forward so many valuable and valuable suggestions!
I have updated the summary message based on your comments and Chase's comments

File: src/init/azext_init/_help.py [hunk 9]
```diff
@@ -8,6 +8,15 @@
 
 
 helps['init'] = """
-    type: command
-    short-summary: It's an effortless setting up tool for configs.
+type: command
+short-summary: It's an effortless setting up tool for configs by offering an interactive question-answer experience
```
Comment (dbradish-microsoft) [inline src/init/azext_init/_help.py:12]:
```suggestion
short-summary: Set Azure CLI user configurations by responding to questions with multiple choice answers.
```

**Review by dbradish-microsoft — CHANGES_REQUESTED** [review]
The majority of our short-summary values are in the active voice with an understood "you".  So "<you> Manage ..." or "<you> Configure...".  Our short-summary values will all be standardized this way in fall 2022 under work item [1953041](https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1953041).

@Chasewilson, please advise if you would prefer a different active-voice phrase.

**Comment by chasewilson** [comment]
> The majority of our short-summary values are in the active voice with an understood "you". So "<you> Manage ..." or "<you> Configure...". Our short-summary values will all be standardized this way in fall 2022 under work item [1953041](https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1953041).
> 
> @chasewilson, please advise if you would prefer a different active-voice phrase.

I believe the docs guidance is that we should use active voice as much as possible and should try and stay consistent.

**Comment by zhoxing-ms** [comment]
@dbradish-microsoft Thank you for your detailed explanation. I agree that the active voice should be used in `short-summary` to keep consistency. 
I have updated the `short-summary`, please help review again, thank you

**Review by jsntcy — APPROVED** [review]

**Review by dbradish-microsoft — APPROVED** [review]

**Pull request merged**